src/WebsockettestDatastructures.js

```diff
@@ -153,14 +153,14 @@ function RMBTTestResult() {
 RMBTTestResult.prototype.addThread = function(rmbtThreadTestResult) {
     this.threads.push(rmbtThreadTestResult);
 };
-RMBTTestResult.prototype.ip_local;
-RMBTTestResult.prototype.ip_server;
-RMBTTestResult.prototype.port_remote;
-RMBTTestResult.prototype.num_threads;
+RMBTTestResult.prototype.ip_local = undefined;
```

Comment:
My IDE was not happy with it, so I chose to be more explicit. But I could change those to `null`?